test/functional/legacy/breakindent_spec.lua

```diff
@@ -129,7 +129,7 @@ describe('breakindent', function()
     feed_command('let g:test="Test 15: breakindent + visual blockwise delete #2"')
     feed_command('%d')
     feed_command('normal! 4a1234567890')
-    feed_command([[exec "normal! >>\<C-V>3f0x"]])
+    feed_command([[exec "normal! >>^\<C-V>3f0x"]])
```

Comment:
this shouldn't be necessary. I think what is happening is, the `set all&` call on line 95 is setting the wrong default. because `options.lua` should set `vim=false`

Reply:
with nostartofline, the cursor remains at the end of the insertion after `normal! 4a1234567890` whereas with startofline, it puts it at the beginning of the line so setting the correct default would also require my modification I think.
if I start the binary, `:verbose set startofline?` returns `nostartofline`. 
I confess I am kinda confused by the values in options.lua, I assume I should have in the end the following behavior:
```
:set startofline&vi -> startofline
:set startofline&vim -> startofline
:set startofline& -> nostartofline
```
?

Reply:
> `:set startofline&vim -> startofline`

? That would contradict the actual default. Although it's debatable, up to now we've updated "vim defaults" to whatever our defaults are. Otherwise we would need to introduce `&nvim` variant for `:set`.

Reply:
> Otherwise we would need to introduce `&nvim` variant for `:set`.

I've thought about that being useful myself before.

Reply:
@blueyed yeah, it's probably a good idea. Then we could use `:set all&vim` as a way of checking "vim defaults" against user reports.